packages/seacas/libraries/ioss/src/exonull/Ioexnl_Utils.C

```diff
@@ -230,44 +230,6 @@ namespace Ioexnl {
     return true;
   }
 
-  void decode_surface_name(Ioexnl::SideSetMap &fs_map, Ioexnl::SideSetSet &fs_set,
```

Comment:
Is this where you removed the tokenization of sideset names? And there is no lingering effect from this?

Reply:
All of the Sierra nightly, regression, and other dashboard tests pass...